src/types/ITbGridProps.ts

```diff
@@ -0,0 +1,18 @@
+import { ColumnModel, DataGridStorage, TubularHttpClientAbstract } from 'tubular-common';
+import { ITbRowProps } from './ITbRowProps';
+
+export interface ITbGridProps {
```

Comment:
Can you avoid the I prefix?

Reply:
I can't, because all the interfaces in this repo are using it. So either we remove it (breaking change) or we continue using it.